src/nodes.coffee

```diff
@@ -3054,7 +3054,7 @@ exports.ClassProperty = class ClassProperty extends Base
       key: @name.ast o, LEVEL_LIST
       value: @value.ast o, LEVEL_LIST
       static: !!@isStatic
-      computed: @name instanceof ComputedPropertyName
+      computed: @name instanceof Index or @name instanceof ComputedPropertyName
```

Comment:
There were a couple remaining cases where `computed` was not being set correctly on class methods/properties (added a test)